### ISSUE TITLE
resize_activation function

### ISSUE THREAD

**Issue by Paseam** [github-pengge/PyTorch-progressive_growing_of_gans#1]
hi，as you say you replace repeat by torch.nn.functional.upsample, but i read the source codes from author, it seems that author uses repeat, do you notice it ?

**Comment by github-pengge** [comment]
However, using repeat suffers from severe perturbation when fading in. Besides, the author used  repeat to implement nearest neighbor upsampling. You can verify that via numpy.repeat(PyTorch can not achieve this through repeat, since it only repeat at size 1 dim.).
